pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -192,8 +195,32 @@ func (r *ReconcileIBMBlockCSI) Reconcile(request reconcile.Request) (reconcile.R
 			err = fmt.Errorf("failed to update IBMBlockCSI CR: %v", err)
 			return reconcile.Result{}, err
 		}
+		return reconcile.Result{}, nil
```

Comment:
Adding this `return` statement means changing the software behavior since it won't execute lines 225-255.
Why do we want to return here?

Reply:
it was part of develop branch, not relevant to this PR, fixed